frontstage/templates/sign-in/sign-in.account-locked.html

```diff
@@ -0,0 +1,38 @@
+{% import 'partials/section.html' as section %}
+{% extends "layouts/_twocol.html" %}
+
+{% block page_title %}Trouble signing in? - ONS Business Surveys{% endblock %}
+
+{% block main %}
+    <h1 class="saturn">Check your email</h1>
+
+    <p>You have tried to sign in few times with the wrong details.</p>
+    <p>If your email is registered to an account, we will send an email explaining what how to sign in.</p>
+    <p>If you do not receive this email in 15 minutes, you can try the following: </p>
```

Comment:
- This should read "You have tried to sign in `a` few times...."
- Need to remove the `what` (i.e. "we will send an email explaining how to sign in")